assets/js/blocks/cart-checkout/cart/checkout-button/style.scss

```diff
@@ -1,3 +1,8 @@
 .wc-block-cart__submit-button {
 	width: 100%;
+	margin: 0 0 $gap;
```

Comment:
This is so there is space between the buttons and the icons, unless there are no icons in which case no margins are needed.